packages/canary-features/addon/default-features.js

```diff
@@ -0,0 +1,9 @@
+// -build-infra/src/features consumes this variable to
+// populate the default features
+// we export it from its own file to avoid
+// requiring @ember/polyfills for the esm require
+export const DEFAULT_FEATURES = {
```

Comment:
Lets use a default export here:

```suggestion
export default {
```